### PR TITLE
Redefine context_data and worker_cls for ClusterRpcProxy + python3 support

### PR DESCRIPTION
**Pull request by scarchik**
I has done some useful changes that need for my project and it will be great if I can commit it to your repository.

- Support Python 3 (python 3.5 tested, python2 support shouldn't be broken)
- Availability to redefine context_data and worker_cls  for ClusterRpcProxy
- Add common exception for flask_nameko

Sorry for so many changes in one pull request.

### CODE REVIEW

File: flask_nameko/proxies.py [hunk 1]
```diff
@@ -31,37 +38,47 @@ def configure(self, config):
...
         self._connect_on_method_call = connect_on_method_call
         if app:
-            self.init_app(app)
+            self.init_app(app, context_data, worker_cls)
 
-    def init_app(self, app):
+    def init_app(self, app, context_data=None, worker_cls=None):
         config = dict()
-        for key, val in app.config.iteritems():
+        for key, val in app.config.items():
```
Comment (jessepollak) [inline flask_nameko/proxies.py:74]: Can we use `six.iteritems` here?

**Review by jessepollak — CHANGES_REQUESTED** [review]
This looks great to me — thank you for the PR! Two quick things:

1) Can you add a `six` dependency and use it in the place I mentioned to maintain compatibility with Python 2?
2) If possible, can you add Python 3.5 to tox so it can be tested?

Thanks again for the PR!

File: flask_nameko/proxies.py (hunk 1 above)
Comment (scarchik) [reply flask_nameko/proxies.py:74]: It looks unimportant: 
- this code use on application start only and shouldn't decrease performance
- config dictionary is not so big
- more one site-package dependency: **six**

If you disagree, I can add, what you think?

Comment (jessepollak) [reply flask_nameko/proxies.py:74]: I'm fine with that - thanks :)

**Comment by jessepollak** [comment]
I'm fine with not including `six`, but can we add `3.5` to `tox`? Thanks!

**Pull request closed without merging**